code/__HELPERS/icons.dm

```diff
@@ -1006,4 +1006,22 @@ var/global/list/humanoid_icon_cache = list()
 //Images have dir without being an atom, so they get their own definition.
 //Lame.
 /image/proc/setDir(newdir)
-	dir = newdir
+	dir = newdir
+
+// Used to make the frozen item visuals for Freon.
+
+/obj/proc/make_frozen_visual(var/obj/F)

```

Comment:
should really drop the ```_visual``` since it actually sets the ```is_frozen``` var, making it REALLY frozen.
also why on earth does this take an argument and not just use ```src```?